tests/fsharp/tests.fs

```diff
@@ -32,9 +32,11 @@ let singleTestBuildAndRun = getTestsDirectory >> singleTestBuildAndRun
 let singleTestBuildAndRunVersion = getTestsDirectory >> singleTestBuildAndRunVersion
 let testConfig = getTestsDirectory >> testConfig
 
-
 [<NonParallelizable; SetUICulture("en-US"); SetCulture("en-US")>]
-module CoreTests =
+module TestsToBeDeletedBecauseTheyHaveMoved =
```

Comment:
So far I was not deleting the tests, I am just moving them from one place to this one.
This module is basically a marker for "can be deleted, because it is migrated over"